.travis.yml

```diff
@@ -31,7 +55,6 @@ branches:
 
 deploy:
   provider: pages
-  skip_cleanup: true
```

Comment:
Why remove this?

Reply:
oh right, I ran the travis linter and  the `skip_cleanup` is added twice in the travis file

https://github.com/Skyscanner/backpack/blob/master/.travis.yml#L34
https://github.com/Skyscanner/backpack/blob/master/.travis.yml#L39

I'm removing one, I guess it's kind of hard to spot that in the diff github shows, good shout 👍